pages/register.js

```diff
@@ -0,0 +1,231 @@
+              <label>
+                <div>Do you have your instrument already?:</div>
+                <Field component="select">
+                  <option>yes</option>
+                  <option>no</option>
+                  <option>I need assistance in purchasing/renting</option>
+                </Field>
+              </label>
+            </div>
+
+            <div>
+              <label>
+                <div>Availabilites:</div>
+                <FieldArray name="availability">
+                  {({ push, remove }) => (
+                    <>
+                      {values.availability &&
+                        values.availability.length > 0 &&
+                        values.availability.map((item, index) => (
+                          <div key={index}>
+                            <Field
+                              component="select"
+                              name={`availability[${index}].day`}
+                              className="app__form_dropdownbox"
+                            >
+                              <option value="">Select</option>
+                              {businessDays.map(element => (
+                                <option key={element.id} value={element.day}>
+                                  {element.day.replace(/^\w/, c =>
+                                    c.toUpperCase()
+                                  )}
+                                </option>
+                              ))}
+                            </Field>
+                            <span />
+                            <Field
+                              component="select"
+                              name={`availability[${index}].fromTime`}
+                            >
+                              <option value="">From</option>
+                              {businessHours.map(element => (
+                                <option key={element.id} value={element.time}>
+                                  {element.label}
+                                </option>
+                              ))}
+                            </Field>
+                            <span />
+                            <Field
+                              component="select"
+                              name={`availability[${index}].toTime`}
+                            >
+                              <option value="">To</option>
+                              {businessHours.map(element => (
+                                <option key={element.id} value={element.time}>
+                                  {element.label}
+                                </option>
+                              ))}
+                            </Field>
+                            <button
+                              type="button"
+                              className="btn"
+                              onClick={() => remove(index)}
+                            >
+                              <i className="far fa-times-circle" />
```

Comment:
Is this indentation expected?

Reply:
We use [Prettier](https://prettier.io/) for the code formatting